README.md

```diff
@@ -1,7 +1,7 @@
 # Use MATLAB with GitLab CI/CD
-You can use [GitLab&trade; CI/CD](https://docs.gitlab.com/ee/ci/index.html) to run MATLAB&reg; scripts, functions, and statements as part of your pipeline. You also can run your MATLAB and Simulink&reg; tests and generate artifacts such as JUnit test results and Cobertura code coverage reports. 
+You can use [GitLab&trade; CI/CD](https://docs.gitlab.com/ee/ci/index.html) to build and test your MATLAB&reg; project as part of your pipeline. For example, you can identify the code issues in your project, run your tests and generate test and coverage artifacts, and package your files into a toolbox.
 
-The `MATLAB.gitlab-ci.yml` [template](https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/MATLAB.gitlab-ci.yml) provides you with jobs that show how to run MATLAB scripts, functions, statements, and tests. To run MATLAB code and Simulink models based on this template, you must use the Docker executor to run MATLAB within a container. The [MATLAB Container on Docker Hub](https://www.mathworks.com/help/cloudcenter/ug/matlab-container-on-docker-hub.html) lets you run your build using MATLAB R2020b or a later release. If your build requires additional toolboxes, use a custom MATLAB container instead. For more information on how to create and use a custom MATLAB container, see [Create a Custom MATLAB Container](https://www.mathworks.com/help/cloudcenter/ug/create-a-custom-matlab-container.html).
+The `MATLAB.gitlab-ci.yml` [template](https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/MATLAB.gitlab-ci.yml) provides you with jobs that show how to run a MATLAB build, as well as MATLAB tests, scripts, functions, and statements. To run MATLAB code and Simulink models based on this template, you must use the Docker executor to run MATLAB within a container. The [MATLAB Container on Docker Hub](https://www.mathworks.com/help/cloudcenter/ug/matlab-container-on-docker-hub.html) lets you run your build using MATLAB R2020b or a later release. If your build requires additional toolboxes, use a custom MATLAB container instead. For more information on how to create and use a custom MATLAB container, see [Create a Custom MATLAB Container](https://www.mathworks.com/help/cloudcenter/ug/create-a-custom-matlab-container.html).
```

Comment:
Do we want to list the jobs in the same order they appear in the template?

Reply:
Good suggestion. Updated.